lib/alerts_viewer_web/live/bus_live.html.heex

```diff
@@ -22,8 +22,11 @@
 </div>
 
 <.list>
-  <:item title="Accuracy">
-    <%= PredictionResults.accuracy(@prediction_results) %>%
```

Comment:
question: Is `PredictionResults.accuracy` still in use anywhere, or can we just delete it since `balanced_accuracy` is preferable?

Reply:
It's no longer being used. I'll go ahead and take out the function from prediction results.